packages/components/image-viewer/src/image-viewer.vue

```diff
@@ -38,7 +38,8 @@
             </span>
           </template>
           <!-- ACTIONS -->
-          <div :class="[ns.e('btn'), ns.e('actions')]">
+          <div :class="ns.e('actions')">
+            <span :class="ns.e('actions__progress')">{{ progress }}</span>
```

Comment:
The original structure should not be changed here. I think it is better to put it at the same level as action.